requirements-dev.txt

```diff
@@ -3,3 +3,4 @@ google-cloud-storage==1.38.0
 pytest==6.2.4
 pytest-cov==2.12.1
 requests==2.25.1
+fs==2.4.13
```

Comment:
Is this needed?

Reply:
not so sure in other environments. but in my local virtual environment (macOS BigSur with Homebrew + vanilla Python 3.9.5), it complains about missing fs package, which is used in `storage.py`.

Reply:
Weird since setup.py should install it. https://github.com/oittaa/gcp-storage-emulator/blob/main/setup.py#L58

Reply:
ah ok. I will remove it.